app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/gainorlossafterreliefs/GainOrLossAfterReliefsController.scala

```diff
@@ -102,13 +102,42 @@ class GainOrLossAfterReliefsController @Inject() (
                 else {
                   val updatedAmount      = AmountInPence.fromPounds(value)
                   val updatedDraftReturn =
-                    draftReturn.fold(
-                      _.copy(gainOrLossAfterReliefs = Some(updatedAmount)),
-                      _.copy(gainOrLossAfterReliefs = Some(updatedAmount)),
-                      _.copy(gainOrLossAfterReliefs = Some(updatedAmount)),
-                      _.copy(gainOrLossAfterReliefs = Some(updatedAmount)),
-                      _.copy(gainOrLossAfterReliefs = Some(updatedAmount))
-                    )
+                    if (fillingOutReturn.isFurtherReturn.contains(true))
```

Comment:
gain or loss after reliefs is always for a further return so you don't need to do this check here - it'll always be true